app/models/decorators/mvi_user_decorator.rb

```diff
@@ -22,10 +22,11 @@ def create
       mvi: response
     }
     @user
-  rescue MVI::RecordNotFound => e
+  rescue MVI::RecordNotFound
     # TODO(AJD): add metric
     Rails.logger.error "MVI record not found for user: #{@user.uuid}"
-    raise Common::Exceptions::RecordNotFound, "User not in found MVI: #{e.message}"
+    @user.attributes = {}
```

Comment:
`#attributes=` is used by [virtus](http://www.rubydoc.info/github/solnic/virtus/Virtus/InstanceMethods/MassAssignment) for mass assignment so I'm pretty sure you can remove this line